test/test-server/org_handlers.go

```diff
@@ -64,6 +64,48 @@ func handleOrgEnvironments(t *testing.T) http.HandlerFunc {
 	}
 }
 
+// Handler for: "/org/v2/organizations/{id}"
+func handleOrgOrganization(t *testing.T) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		vars := mux.Vars(r)
+		id := vars["id"]
+		w.Header().Set("Content-Type", "application/json")
+
+		displayName := "default"
+		switch r.Method {
+		case http.MethodPatch:
+			req := orgv2.OrgV2Environment{}
+			err := json.NewDecoder(r.Body).Decode(&req)
```

Comment:
More idiomatic; if you were to pass this variable into 10 different functions, it would be easier not to have to add `&` every time.
```suggestion
			req := &orgv2.OrgV2Environment{}
			err := json.NewDecoder(r.Body).Decode(req)
```